mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,404 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ..constants import FIFF
+from ..utils import _mult_cal_one
+from ..meas_info import _empty_info
+from ..base import _BaseRaw
+from ..utils import _read_str
+
+
+def read_raw_cnt(input_fname, eog=(), ecg=(), emg=(), misc=(),
```

Comment:
can we try to keep the uniformity of the constructor signature similar across the EEG modules, they all have the same beginning for the constructor, `filename, montage, eog, misc`. It keeps a consistent look across these modules